<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_In</class>
 <widget class="QWidget" name="Main_In">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QLabel" name="carNum_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车牌图片：</string>
    </property>
   </widget>
   <widget class="QLabel" name="inTime_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>入场时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="carNum2_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>车牌号码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="video_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>671</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="carNum_image">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyNum_butt">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carNum_butt">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>130</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>车牌识别</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>290</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="carNum_edit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="inTime_edit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
